exercises/APE5-6/APE05-06E1a.tex

```diff
@@ -0,0 +1,118 @@
+To Find all Nash equilibria in the following games, we have to check all the support. Here we have 9 support where 4 are pure strategies. For all choices of support D $= (D_i)_{i\in N}$ we have to verify the 3 conditions (see syllabus p40).
+
+\begin{itemize}
+  \item[$\bullet$] Pure strategy supports. 
+\end{itemize}
+
+- $\mathbf{\{x_1\}}\times\mathbf{\{x_2\}}$.
+
+The payoff of the players are $w_1=2$ and $w_2=1$. It's easy to see that if the player 2 play $y_2$ instead of $x_2$ we have :
+\begin{equation*}
+    1=w_2<u_2(x_1,y_2)=2
+\end{equation*}
+This means that the third equation does not hold, so this pure strategy is not a equilibrium 
+
+- $\mathbf{\{x_1\}}\times\mathbf{\{y_2\}}$.
+
+The payoff of the players are $w_1=1$ and $w_2=2$. It's easy to see that if the player 1 play $y_1$ instead of $x_1$ we have :
+\begin{equation*}
+    1=w_1<u_1(y_1,y_2)=2
+\end{equation*}
+This means that the third equation does not hold, so this pure strategy is not a equilibrium 
+
+- $\mathbf{\{y_1\}}\times\mathbf{\{x_2\}}$.
+
+The payoff of the players are $w_1=1$ and $w_2=5$. It's easy to see that if the player 1 play $x_1$ instead of $y_1$ we have :
+\begin{equation*}
+    1=w_1<u_1(x_1,x_2)=2
+\end{equation*}
+This means that the third equation does not hold, so this pure strategy is not a equilibrium 
+
+- $\mathbf{\{y_1\}}\times\mathbf{\{y_2\}}$.
+
+The payoff of the players are $w_1=2$ and $w_2=1$. It's easy to see that if the player 2 play $x_2$ instead of $y_2$ we have :
+\begin{equation*}
+    1=w_1<u_1(y_1,x_2)=5
+\end{equation*}
+This means that the third equation does not hold, so this pure strategy is not a equilibrium 
+
+
+\begin{itemize}
+  \item[$\bullet$] Randomized vs pure strategy supports : This corresponds to the case when for example the first player randomizes between $x_1$ and $y_1$, while the second sticks either to $x_2$ or to $y_2$.
+\end{itemize}
+
+- $\mathbf{\{x_1,y_1\}}\times\mathbf{\{x_2\}}$.
+
+The second equation tell us that 
```

Comment:
Be clearer here. Say something like "By the Payoffs equation of Procedure~2, we must have"